src/App.css

```diff
@@ -28,7 +28,35 @@ ul {
   }
 }
 
-
 .reviewsList__img {
   height: 270px;
+  border-radius: 15px;
```

Comment:
Really like the styling you have in place here 😄 Is this app responsive in mobile view? Not essential right now, but something to consider later on.